app/Http/Controllers/ProfileController.php

```diff
@@ -331,7 +333,7 @@ public function update(Request $request, $rname)
 
                 // Add skills, reactivate deleted ones if they exist
                 foreach ($added as $skill) {
-                    $prev_deleted = $knight->skills()->wherePivot('delflg', true)->find($skill);
+                    $prev_deleted = $knight->skills(true)->find($skill);
```

Comment:
I'd prefer explicity using the named argument in situations like this one to make the code more readable.

Reply:
That's fair, added the name of the parameter.